main.tf

```diff
@@ -0,0 +1,189 @@
+/**
+* ## Module: cloudposse/terraform-aws-lambda-elasticsearch-cleanup
+*
+* This module creates a scheduled Lambda function which will delete old
+* Elasticsearch indexes using SigV4Auth authentication. The lambda 
+* function can optionally send output to an SNS topic if the topic ARN
+* is given
+*/
+
+# Terraform
+#--------------------------------------------------------------
+terraform {
+  required_version = ">= 0.10.2"
+}
+
+# Data
+#--------------------------------------------------------------
+data "aws_iam_policy_document" "assume_role" {
+  statement {
+    actions = ["sts:AssumeRole"]
+    effect  = "Allow"
+
+    principals {
+      type        = "Service"
+      identifiers = ["lambda.amazonaws.com"]
+    }
+  }
+}
+
+data "aws_iam_policy_document" "default" {
+  statement {
+    actions = [
```

Comment:
do we need to give these permissions to the module?

Reply:
Yeah these are needed to the Lambda function can a) write log files to CloudWatch logs b) post a v4 signed payload to ES